### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by xiexr151e**
I would like my code and the README.md to be checked for styling.
This README.md is a work in progress; I have only written up instructions with `psi4`, and I plan to add additional info regarding `TensorMol` later.
As for the code itself, I would like to be checked for the documentation of each function.

One concern I have is being able to create other scripts by importing this script.
For example, I want to be able to do something along the lines of "import [name of script here]" as a package.

@agoetz

### CODE REVIEW

File: qm_mb_energy_calculator/README.md [hunk 1]
```diff
@@ -1,14 +1,90 @@
...
+
+## About `settings.ini`:
+
+Use this file to customize your options to use this program. This file must be in the same directory as your current working directory.
+
+#### [driver]: Settings pertaining to the initialization of the program.
+* input: The name of the xyz input file, including its file extension.
+* model: The calculation model we are using for calculations. Currently only supports `psi4`.
+* overwrite: Determines whether we wish to overwrite an output file, if one already exists.
+* append: Determines whether we wish to append to an output file, if one already exists.
```
Comment (xiexr151e) [inline qm_mb_energy_calculator/README.md]: Overwrite and append are currently unused flags; I plan to implement them next week.

File: qm_mb_energy_calculator/README.md [hunk 2]
```diff
@@ -1,14 +1,90 @@
...
+This program can be used to calculate the many-body decomposition energies, as well as the k-mer energies of a molecule.
+These molecules can also be defined as fragments by the user, inside an xyz input file.
 
-Inside the `scripts` directory, `run_psi4_1B.py` uses `pybel` to read in a one-body xyz file and generate a one-body training set using PSI4. PSI4 output files are placed in a `calculations` directory within the current directory and are compressed into a zip file.
- 
-In the future, this will contain a script(s) to read in geometries and calculate 1B, 2B, 3B, and NB reference energies using PSI4. 
+## To use this program:
 
-References:
+### Setup:
```
Comment (xiexr151e) [inline qm_mb_energy_calculator/README.md]: Andy mentioned the need of a requirements section, but I'm not sure if I am splitting this correctly.

File: qm_mb_energy_calculator/README.md [hunk 3]
```diff
@@ -1,14 +1,91 @@
...
+To begin, it is assumed that the user has installed a `conda` build of some kind.
 
-1.[Noel M. O’Boyle, Michael Banck, Craig A. James, Chris Morley, Tim Vandermeersch, Geoffrey R. Hutchison. Open Babel: An open chemical toolbox. J. Cheminf. 2011, 3, 33.](https://jcheminf.springeropen.com/articles/10.1186/1758-2946-3-33)
+To start, create a `conda` environment that includes Python 3, and TensorFlow 1.4.0 (**This is important! Installing a newer version of TensorFlow will cause errors!**). The command is as follows:
```
Comment (zonca) [inline qm_mb_energy_calculator/README.md]: do you need Tensorflow to run `psi4`?

File: qm_mb_energy_calculator/README.md [hunk 4]
```diff
@@ -1,14 +1,91 @@
...
+These molecules can also be defined as fragments by the user, inside an xyz input file.
 
-Inside the `scripts` directory, `run_psi4_1B.py` uses `pybel` to read in a one-body xyz file and generate a one-body training set using PSI4. PSI4 output files are placed in a `calculations` directory within the current directory and are compressed into a zip file.
- 
-In the future, this will contain a script(s) to read in geometries and calculate 1B, 2B, 3B, and NB reference energies using PSI4. 
+## To use this program:
 
-References:
+### Setup:
+To begin, it is assumed that the user has installed a `conda` build of some kind.
```
Comment (zonca) [inline qm_mb_energy_calculator/README.md]: I think it is simpler to write: "has installed Anaconda"

File: qm_mb_energy_calculator/README.md [hunk 5]
```diff
@@ -1,14 +1,91 @@
...
+#### [psi4]: Settings specific to the `psi4` model.
+* memory: The memory `psi4` will use for calculation.
+* method: The method `psi4` will use for calculation.
+* basis: The basis set `psi4` will use for calculation.
+* charge: Currently unused.
+* spin: Current unused.
+* threads: The amount of parallel threads `psi4` will use for calculation.
+
+
+#### [MBdecomp]: Settings pertaining many-body analysis of the molecule.
```
Comment (zonca) [inline qm_mb_energy_calculator/README.md]: I don't know what decomposition means, anyway, can we split it into another folder and only keep psi4 here?

File: qm_mb_energy_calculator/README.md [hunk 6]
```diff
@@ -1,14 +1,91 @@
-#### Training Set Generation
-This directory contains the scripts to read in geometries using Open Babel's `pybel` and calculates their 1B energies using [PSI4](http://www.psicode.org), an open-source suite of ab initio quantum chemistry programs designed for efficient, high-accuracy simulations of a variety of molecular properties.
+# Training Set Generation
 
-Open Babel's `openbabel` module provides direct access to the C++ Open Babel library from Python [1]. The module comes with the `pybel` wrapper which provides convenience functions and classes that make it simpler to use the Open Babel libraries from Python, especially for file input/output and for accessing the attributes of atoms and molecules [2].  
+## Overview:
+This program can be used to calculate the many-body decomposition energies, as well as the k-mer energies of a molecule.
```
Comment (zonca) [inline qm_mb_energy_calculator/README.md]: we need a simpler explanation that even a non-expert can understand on what is the purpose of this code and who is expected to use it

**Comment by agoetz** [comment]
Closing this pull request as it is outdated. Changes have been incorporated into newer pull requests.

**Pull request closed without merging**